build/Build.cs

```diff
@@ -291,6 +296,45 @@ public Build()
                           CopyFile(file, LocalPackagesDirectory / Path.GetFileName(file), FileExistsPolicy.Overwrite);
                   });
 
+        Target CopySashimiPackagesForConsolidation =>
+            _ => _.DependsOn(Compile)
+                 .Executes(() =>
+                           {
+                               foreach (var file in Directory.GetFiles(SashimiPackagesDirectory))
+                                   CopyFile(file, ArtifactsDirectory / Path.GetFileName(file), FileExistsPolicy.Overwrite);
+                           });
+
+        Target PackageConsolidatedCalamariZip =>
+            _ => _.DependsOn(PackBinaries)
+                  .DependsOn(CopySashimiPackagesForConsolidation)
+                  .Executes(() =>
+                            {
+                                var artifacts = Directory.GetFiles(ArtifactsDirectory, "*.nupkg");
```

Comment:
I noticed that right now, _everything_ is being published as a NuPkg, including things like `Calamari.Common`, which don't contain a Calamari executable. The Consolidate task is picking these up, and adding them unnecessarily to the consolidated package. 

Could we use an inclusion-list to only consider a subset of packages known to contain Calamari executables here?

Reply:
Yup that seems more appropriate - I'll follow this up in another PR to have optimisations and tests in place.